source/renderer/app/components/profile/terms-of-use/TermsOfUseText.scss

```diff
@@ -4,7 +4,10 @@
   user-select: text;
 
   a {
-    color: var(--theme-support-settings-text-color);
+    border-bottom: 1px solid var(--rp-link-color);
```

Comment:
@DeeJayElly please introduce new css var for this: `--theme-support-settings-link-color`

Reply:
It appears that variable already exists, so just added it instead --rp-link-color.